src/state/utils.js

```diff
@@ -2,27 +2,41 @@ import { createAsyncThunk as createAsyncThunkBase } from '@reduxjs/toolkit'
 
 import { addMessage } from 'notifications/notificationsSlice'
 import { signOut } from 'account/accountSlice'
+import { refreshToken } from 'account/auth'
 import { UNAUTHENTICATED } from 'account/authConstants'
 
+const executeAuthRequest = (dispatch, action) => action().catch(async error => {
+  if (error !== UNAUTHENTICATED) {
+    // If not auth error return request error
```

Comment:
nitpick: the `if` statement itself is pretty clear on its meaning. Perhaps this comment is unnecessary.